declarations/Rephrase.ai.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Rephrase.ai",
+  "documents": {
+    "Human Rights Policy": {
```

Comment:
These terms are relevant to track but I believe we don't have a matching terms type for the moment!

Reply:
Yes I agree, this might need a new category type!